lemur/tests/test_endpoints.py

```diff
@@ -32,7 +32,7 @@ def test_rotate_certificate(client, source_plugin):
 )
 def test_endpoint_get(client, token, status):
     assert (
-        client.get(api.url_for(Endpoints, endpoint_id=2), headers=token).status_code
+        client.get(api.url_for(Endpoints, endpoint_id=999), headers=token).status_code
```

Comment:
I changed this because the test started returning 200 instead of 404; I'm assuming some other test changes I made (that add more endpoints) caused ID 2 to be valid.